routing/localchans/manager.go

```diff
@@ -180,6 +248,85 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 	return failedUpdates, nil
 }
 
+// createEdge recreates an edge and policy from an open channel in-memory.
+func (r *Manager) createEdge(channel *channeldb.OpenChannel) (
+	*models.ChannelEdgeInfo, *models.ChannelEdgePolicy, error) {
+
+	var (
+		nodeKey1Bytes, nodeKey2Bytes,
```

Comment:
Nit: Define all variables for the Local = true case, and only in the if below switch it if false ? (Its a nit but I think style wise looks better)

Reply:
Done, let me know what you think (I think it looks nicer indeed)

Reply:
I think its easier like this:

```
if byte.Compare ...
nodeKey1Bytes, nodeKey2Bytes = nodeKey2Bytes, nodeKey1Bytes
bitcoinKey1Bytes, bitcoinKey2Bytes = bitcoinKey2Bytes, bitcoinKey1Bytes
```

Reply:
amazing